### DISCUSSION TITLE
TikTok-style toon vid

### DISCUSSION THREAD

**Discussion by AfroCodec** [kodadot/nft-gallery#3259] [KodAmbassadors & community stuff]
https://user-images.githubusercontent.com/37082587/175466404-72966507-6e9d-447c-93de-30704d6d3b9f.mp4

UPDATED: Instagram Feed format (aspect ratio 4:5)   https://drive.google.com/file/d/16Po9N_sbhaUzjPNKuZly2BsrsQc_GqzU/view?usp=sharing

Should be good for Twitter also 

Videos are a great way to convey messages and ideas. As a KodaDot ambassador, I'd be using my little toon skills to create fun, explanatory and colourful videos that'd promote KodaDot.
Here's a TikTok-style toon vid I made. 😊

**Comment by damskyftw** [comment]
Can you also make it in twitter format? I mean why not put it there aswell?

Also in instagram would be perfect!

**Comment by DeluneClaire** [comment]
This is great! I love it 🔥

**Comment by DeluneClaire** [comment]
![bad size ig](https://user-images.githubusercontent.com/104791453/177342558-f7da2aa5-31b7-4351-be9e-107850e016fe.jpeg)
![Koda video bad size](https://user-images.githubusercontent.com/104791453/177342571-470dea8f-676f-4bb1-85ba-e9c6f445f29d.jpeg)

I tried to post it on IG, but it's too high, so the Koda logo doesn't appear. Could you make it in this size: 

Aspect ratio: 9:16
Recommended upload size: 1080 pixels x 1920 pixels.

**Reply by AfroCodec** [reply]
Hi, @DeluneClaire . The video is already in the 1080x1920 pixels resolution, and aspect ratio 9:16; this resolution is perfect for Instagram Reels and TikTok. 
Seems Instagram auto crops its Feed portrait videos to aspect ratio 4:5. I have created a 4:5 aspect ratio of the video -- link to file below 
Because of GitHub size limits, I uploaded it to Google Drive

https://drive.google.com/file/d/16Po9N_sbhaUzjPNKuZly2BsrsQc_GqzU/view?usp=sharing

**Reply by AfroCodec** [reply]
I think the 4:5 aspect ratio would be good for Twitter also; but you could also suggest to me which resolution/aspect ratio you'd prefer for Twitter

**Comment by DeluneClaire** [comment]
Yes, this size you send me is just fine for Instagram. Thank you :)

**Comment by AfroCodec** [comment]
Hence forth I'd be doing 2 or 3 resolutions for videos — 1080 × 1920 for TikTok and IG reels; 
1080 × 1350 with aspect ratio 4:5 for IG feed; 
then maybe 720 × 720, aspect ratio 1:1, square resolution for Twitter. 

For optimum viewing experience on all platforms 🚀☺️

**Comment by AfroCodec** [comment]
I read 720p is one of the best res for Twitter video content. I made it in 720p also

https://user-images.githubusercontent.com/37082587/177691689-8e36510f-462e-4429-80e4-40e433e877d5.mp4
